airbrakes/mock/mock_imu.py

```diff
@@ -58,10 +66,18 @@ def __init__(
         # test, because we read the file much faster than update(), sometimes resulting thousands
         # of data packets in the queue, which will obviously mess up data processing calculations.
         # We limit it to 15 packets, which is more realistic for a real flight.
-        self._data_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue(
-            MAX_QUEUE_SIZE if real_time_simulation else SIMULATION_MAX_QUEUE_SIZE
-        )
-
+        if sys.platform == "win32":
```

Comment:
hmm I feel like I've seen this code reused in a few places. Is there anyway to make this a `utils` function?

Reply:
nope, that wouldn't be clean since each time we are doing something different inside the `if` block depending on the situation